CHANGES.md

```diff
@@ -1,15 +1,20 @@
-# next -- TBA
+# Version 1.2 -- 2024-08-30
 
 ## New Features
 
 * Add `mir_str_slice_value` and `mir_str_slice_range_value` functions, which
   allow taking `&str` slices. For more information, see the documentation in the
   [SAW manual](https://github.com/GaloisInc/saw-script/blob/master/doc/manual/manual.md#string-slices).
 
+* Add `mir_mux_values` command for muxing two MIR values.
+
+* Error messages have been improved such that the locations printed with type errors and other diagnostics now have a much stronger connection with reality.
```

Comment:
I would leave this under bug fixes (also, if the idea was to say more than what I'd originally written, e.g. the other fixes to type error printing, that doesn't come across with this wording)